autogen/lua_definitions/functions.lua

```diff
@@ -9340,26 +9341,30 @@ end
 --- @param y number
 --- @param z number
 --- @return number
+--- Finds the height of the highest floor below a given position (x, y, z). If no floor is found, returns the default floor height of -11000.
```

Comment:
same here, but with "`gLevelValues.floorLowerLimit` (default -11000)"

Reply:
Fixed this